mmv1/templates/terraform/examples/cloudquotas_quota_preference_basic.tf.erb

```diff
@@ -0,0 +1,15 @@
+# [START cloudquotas_quota_preference_basic]
+resource "google_cloud_quotas_quota_preference" "<%= ctx[:primary_resource_id] %>" {
+  name          = "projects/<%= ctx[:test_env_vars]['project'] %>/locations/global/quotaPreferences/<%= ctx[:vars]['quota_preference_id'] %>"
+  quota_config  {
+    preferred_value = 210
+  }
+  dimensions = {
+    region = "us-central2"
+  }
+  service       = "compute.googleapis.com"
+  quota_id      = "CPUS-per-project-region"
+  contact_email = "liulola@google.com"
```

Comment:
These examples are made public on our documentation website, so I would consider using a fake email address here.

Reply:
Got it, switched to a fake email.